source/includes/auto-encryption-options.rst

```diff
@@ -0,0 +1,128 @@
+The following table describes the structure of an
+``AutoEncryptionOptions`` object:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Name
+     - Description
+
+   * - **keyVaultClient**
+     - **Type:** ``MongoClient``
+
+       **Description**
+       A ``MongoClient`` configured to connect to
+       the MongoDB instance hosting your {+key-vault-long+}. For more
+
+       If you omit the ``keyVaultClient`` option, the driver uses the
+       MongoDB instance specified to your ``MongoClient`` containing the
+       ``AutoEncryptionOpts`` configuration as the host of your key vault
```

Comment:
Suggestion: Since you have a source constant key-vault-long, use it for all instances or else it may generate maintenance issues in the future.